deploy/1-deploy.ts

```diff
@@ -2,14 +2,24 @@ import hre, { ethers } from "hardhat";
 import { DeployFunction } from "hardhat-deploy/types";
 import { HardhatRuntimeEnvironment } from "hardhat/types";
 
-const func: DeployFunction = async ({ getNamedAccounts, deployments }: HardhatRuntimeEnvironment) => {
+import { ADDRESS_ONE, multisigs, wBNBAddress } from "./../helpers/utils";
+
+const func: DeployFunction = async ({ getNamedAccounts, deployments, network }: HardhatRuntimeEnvironment) => {
   const { deploy } = deployments;
   const { deployer } = await getNamedAccounts();
 
-  const vBNBAddress = (await ethers.getContract("vBNB")).address;
-  const comptrollerAddress = (await ethers.getContract("Unitroller")).address;
-  const WBNBAddress = (await ethers.getContract("WBNB")).address;
-  const timelockAddress = (await ethers.getContract("NormalTimelock")).address;
+  const vBNB = await ethers.getContractOrNull("vBNB");
+  const vBNBAddress = vBNB ? vBNB.address : ADDRESS_ONE;
+
+  const coreComptroller = await ethers.getContractOrNull("Unitroller");
+  const comptrollerAddress = coreComptroller ? coreComptroller.address : ADDRESS_ONE;
+
+  // There is no deployment of WBNB, so how does this work?
+  //const WBNBAddress = (await ethers.getContract("WBNB")).address;
```

Comment:
```suggestion
```
I think we can remove this comment.

Reply:
It worked because we have the deployment info for WBNB in the core pore: https://github.com/VenusProtocol/venus-protocol/blob/develop/deployments/bscmainnet/WBNB.json

At the end, you are doing something similar (using the WBNB address defined in the core repo). I think we could hardcode in this repo (in the `utils.ts` file) the addresses of the WBNB token on BNB chain and BNB testnet. Up to you